CONTRIBUTING.md

```diff
@@ -1,68 +1,84 @@
 # Contributing
 
-## Reporting Issues
+## General Guideline
+
+### Reporting Issues
 
 If you have found what you think is a bug, please [start a discussion](https://github.com/pmndrs/jotai/discussions/new).
 
-Also for usage questions, please [start a discussion](https://github.com/pmndrs/jotai/discussions/new).
+For any usage questions, please [start a discussion](https://github.com/pmndrs/jotai/discussions/new).
 
-## Suggesting new features
+### Suggesting New Features
 
 If you are here to suggest a feature, first [start a discussion](https://github.com/pmndrs/jotai/discussions/new) if it does not already exist. From there, we will discuss use-cases for the feature and then finally discuss how it could be implemented.
```

Comment:
```suggestion
If you are here to suggest a feature, first [start a discussion](https://github.com/pmndrs/jotai/discussions/new?category=ideas) if it does not already exist. From there, we will discuss use-cases for the feature and then finally discuss how it could be implemented.
```